### ISSUE TITLE
Allow to display frendly date range string

### ISSUE THREAD

**Issue by evorios** [gf3/moment-range#67]
It would be nice if it was like this:
```
var range = moment().range(moment("03/15/2015 10:15 AM"), moment("03/26/2015 2:35 PM"));
return range.humanize("LL");
```
Code return: "March 15-26, 2015"

```
var range = moment().range(moment("03/15/2015 10:15 AM"), moment("03/15/2015 02:35 PM"));
return range.humanize("LLL");
```
Code return: "March 15 2015 10:15 a.m. - 2:35 p.m."

```
var range = moment().range(moment("03/15/2015 10:15 AM"), moment("03/15/2018 02:35 PM"));
return range.humanize("YYYY");
```
Code return: "2015 - 2018"

```
var range = moment().range(moment("03/15/2015 10:15 AM"), moment("04/26/2015 2:35 PM"));
return range.humanize("MMMM D");
```
Code return: "March 15 - April 26"

**Comment by husayt** [comment]
Also, will be nice to show human readable periods, which takes into accounts the date context as below.

for a date range  5-feb-2014 and 4-apr-2014  would display `2 months`
5-feb-2014 and 5-apr-2014  would display `2 months and 1 day`
5-feb-2014 and 3-apr-2014  would display `1 months and 30 days`
5-feb-2014 and 3-may-2014  would display `2 months and 29 days`
5-feb-2014 and 3-mar-2014  would display `27 days`
5-feb-2016 and 3-mar-2016  would display `28 days`
and etc

**Comment by gf3** [comment]
@evorios @husayt hey guys, moment-range can now be converted to an ISO 8601 time interval string. and it can also be instantiated from a time interval string as well. this was implemented for #66 

i'll leave the human-readable and localized formatting to the end user to keep this library light. thanks!
